### ISSUE TITLE
How did you manage to bench so high?

### ISSUE THREAD

**Issue by dbhavasagar** [SiddheshNan/Acer-A515-51G-Hackintosh#23]
I have a similar acer laptop and I managed 4002, 11999 on cpu and 29516 on gpu on windows 10, and on osx its around 5%-10% less consistantly.

Also I managed to install Open core 0.5.8 on this and everything seems to work brightness keys nvram boot audio, also no need oo sleep patch for audio, On your clover efi HEVC encode isn't working.  

I'll share my EFI if you want.

Also thanks for the inspiration by making clover for this laptop.

**Comment by SpiGAndromeda** [comment]
Yes! Please share the OpenCore config! I was desprate trying this this weekend! <3

**Comment by dbhavasagar** [comment]
Ok, how much are your Geekbench, Cinebench scores both cpu and compute on Hackintosh?

In clover everything is working and in OpenCore card reader is not working barring the card reader OpenCore is neat and clean than clover, boots times are similar.

I'll upload both if you want.

**Comment by SpiGAndromeda** [comment]
I didn't got OpenCore running.

**Comment by dbhavasagar** [comment]
IAM still testing opencore version, clover works but I don't like it as it don't support SIP feature, I am progressing on open core I will share with you as soon as I get this working well.
Currently IAM trying to get track pad work.

**Comment by SpiGAndromeda** [comment]
@dbhavasagar could you upload it unfinished? I cant even get my Laptop past early boot.
Maybe I could also improve it.

**Comment by dbhavasagar** [comment]
See my profile to download from repo, please report if it's working or not. No need to make any changes just copy and boot.

**Comment by dbhavasagar** [comment]
I updated refi after realising that hdmi wasn't working now it's working and I updated repo

**Comment by dbhavasagar** [comment]
Please tell if working or not, If you install windows you can also use bootcamp drivers to set startup disk.
